api/v2.0/swagger.yaml

```diff
@@ -2343,6 +2343,320 @@ paths:
           description: The API server is alive
           schema:
             type: string
+  /retentions/metadatas:
```

Comment:
Please add `operationId` for these retention APIs just like the method name in the beego handler.

`get:GetMetadatas` may change to `getRentenitionMetadata`

```
	beego.Router("/api/"+version+"/retentions/metadatas", &api.RetentionAPI{}, "get:GetMetadatas")
	beego.Router("/api/"+version+"/retentions/:id", &api.RetentionAPI{}, "get:GetRetention")
	beego.Router("/api/"+version+"/retentions", &api.RetentionAPI{}, "post:CreateRetention")
	beego.Router("/api/"+version+"/retentions/:id", &api.RetentionAPI{}, "put:UpdateRetention")
	beego.Router("/api/"+version+"/retentions/:id/executions", &api.RetentionAPI{}, "post:TriggerRetentionExec")
	beego.Router("/api/"+version+"/retentions/:id/executions/:eid", &api.RetentionAPI{}, "patch:OperateRetentionExec")
	beego.Router("/api/"+version+"/retentions/:id/executions", &api.RetentionAPI{}, "get:ListRetentionExecs")
	beego.Router("/api/"+version+"/retentions/:id/executions/:eid/tasks", &api.RetentionAPI{}, "get:ListRetentionExecTasks")
	beego.Router("/api/"+version+"/retentions/:id/executions/:eid/tasks/:tid", &api.RetentionAPI{}, "get:GetRetentionExecTaskLog")
```